<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QDialog" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>901</width>
     <height>541</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="btnExplorar">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>410</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>EXPLORAR</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMisAmigos">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>410</y>
       <width>231</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>MIS AMIGOS</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblYo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>251</width>
       <height>251</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Recursos.qrc">:/EXTRAS/EXTRAS/userdesc.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCerrarSesion">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>20</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cerrar sesión</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblNombreUsu">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>90</y>
       <width>471</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblApellido">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>150</y>
       <width>471</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblNacimiento">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>210</y>
       <width>481</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblCantAmigos">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>270</y>
       <width>471</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="lblFake_foto">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>60</y>
       <width>231</width>
       <height>231</height>
      </rect>
     </property>
     <property name="text">
      <string>pa no perderlo</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblFake_nombre">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>391</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nombre</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblFake_trabajo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>411</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Trabajo</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblFake_estado">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>421</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Estado</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblFake_hobby">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>431</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Hobby</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblFake_edad">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>431</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Edad</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblNumero_fake">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>421</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Numero:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnFake_next">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>420</y>
       <width>261</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>NO ME INTERESAS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnFake_agregar">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>420</y>
       <width>251</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>¡SEAMOS AMIGOS!</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMiPerfilFake">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>MI PERFIL</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnReactionLike">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>300</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../Pictures/ultimoooo/registro/funcionma/EXTRAS/like.png</normaloff>
       <normalon>:/EXTRAS/EXTRAS/like.png</normalon>../../Pictures/ultimoooo/registro/funcionma/EXTRAS/like.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btnReactionHeart">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>300</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../Pictures/ultimoooo/registro/funcionma/EXTRAS/love.png</normaloff>
       <normalon>:/EXTRAS/EXTRAS/love.png</normalon>../../Pictures/ultimoooo/registro/funcionma/EXTRAS/love.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnReactionSmile">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>300</y>
       <width>80</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../Pictures/ultimoooo/registro/funcionma/EXTRAS/risa.png</normaloff>
       <normalon>:/EXTRAS/EXTRAS/risa.png</normalon>../../Pictures/ultimoooo/registro/funcionma/EXTRAS/risa.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>40</height>
      </size>
     </property>
     <property name="shortcut">
      <string notr="true"/>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QPushButton" name="lblAmigoSiguiente">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>460</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SIGUIENTE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="lblAmigoVolver">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>460</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>VOLVER</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblAmigoFoto">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>201</width>
       <height>221</height>
      </rect>
     </property>
     <property name="text">
      <string>FOTO</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblAmigos">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>10</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>MIS AMIGOS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="lblAmigoEliminar">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>460</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ELIMINAR</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblAmigoNombre">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>100</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>NOMBRE</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblAmigoTrabajo">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>140</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TRABAJO</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblAmigoEstado">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>180</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ESTADO</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblAmigoEdad">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>220</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>EDAD</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblAmigoNumero">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>260</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>NUMERO</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblAmigoHobby">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>310</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>HOBBY</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAmigoMiPerfil">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>MI PERFIL</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listAmigos">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>80</y>
       <width>301</width>
       <height>451</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cboxOrden">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>50</y>
       <width>151</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Nombre</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Numero</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Edad</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Estado</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hobby</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Trabajo</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>40</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Ordenar por:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEnviarMensaje">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>390</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Enviar Mensaje</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QTextEdit" name="textEditMensajes">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>481</width>
       <height>371</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineMensaje">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>390</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSend">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>390</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ENVIAR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="VolverChat">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>420</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Volver</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>20</y>
       <width>371</width>
       <height>421</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Recursos.qrc">:/EXTRAS/EXTRAS/LOGODIMLIM.jpg</pixmap>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
